terra-app-helm/aou-rstudio-chart/templates/deployment.yaml

```diff
@@ -0,0 +1,138 @@
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          securityContext:
+            capabilities:
+              add:
+              - NET_ADMIN
+          command:
+            {{- range .Values.image.command }}
+            - {{ . | quote }}
+            {{- end }}
+          args:
+            {{- range .Values.image.args }}
+            - {{ . | quote }}
+            {{- end }}
+          env:
+            {{- range .Values.extraEnv }}
+            - name: {{ .name }}
+              value: {{ .value | quote }}
+            {{- end }}
+          ports:
+            - name: http
+              containerPort: {{ .Values.image.port }}
+              protocol: TCP
+          volumeMounts:
+            - mountPath: "{{ .Values.persistence.mountPath -}}"
+              name: {{ include "app.fullname" . }}-storage
+          livenessProbe:
+            httpGet:
+              path: "{{ .Values.image.baseUrl }}"
+              port: http
+          readinessProbe:
+            httpGet:
+              path: "{{ .Values.image.baseUrl }}"
+              port: http
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+        - name: welder
+          image: "us.gcr.io/broad-dsp-gcr-public/welder-server:6648f5c"
+          imagePullPolicy: "IfNotPresent"
+          env:
+            - name: CLOUD_PROVIDER
+              value: "gcp"
+            - name: LOCKING_ENABLED
+              value: "true"
+            - name: SHOULD_BACKGROUND_SYNC
+              value: "true"
+            - name: WSM_URL
+              value: "dummy"
+            - name: WORKSPACE_ID
+              value: "dummy"
+            - name: STORAGE_CONTAINER_RESOURCE_ID
+              value: "dummy"
+            - name: STAGING_STORAGE_CONTAINER_RESOURCE_ID
+              value: "dummy"
+            - name: WORKING_DIRECTORY
+              value: "{{ .Values.persistence.mountPath -}}"
+            - name: PORT
+              value: "8080"
+#            - name: GOOGLE_PROJECT
+#              value: "leo provided"
+#            - name: STAGING_BUCKET
+#              value: "leo provided"
+#            - name: SHOULD_BACKGROUND_SYNC
+#              value: "leo provided"
+#            - name: CLUSTER_NAME
+#              value:"leo provided"
+#            - name: RUNTIME_NAME
+#              value: "leo provided"
+#            - name: OWNER_EMAIL
+#              value: "leo provided"
+            {{- range .Values.welder.extraEnv }}
+            - name: {{ .name }}
+              value: {{ .value | quote }}
+            {{- end }}
+          ports:
+            - name: http
+              containerPort: 8080
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: \
+              port: http
+          readinessProbe:
+            httpGet:
+              path: \
+              port: http
+          volumeMounts:
+            - mountPath: "{{ .Values.persistence.mountPath -}}"
```

Comment:
I _think_ welder wants the `mountPath` to be `/work` not `/home/rstudio`: https://github.com/DataBiosphere/welder/blob/6648f5cc6ff170b7413c2561e3a8eb1bce76ab6c/server/src/main/resources/reference.conf#LL6C1-L6C1